### PR TITLE
Fix missing Developer mode requirement for Chocolatey install

### LINKED ISSUE #230: chocolatey installation does not work
I've tried this with admin power shell on two different Windows 10 x64 machines. Here is the full output:

```
choco install fluent-terminal
Chocolatey v0.10.12
2 validations performed. 1 success(es), 1 warning(s), and 0 error(s).

Validation Warnings:
 - A pending system reboot request has been detected, however, this is
   being ignored due to the current Chocolatey configuration.  If you
   want to halt when this occurs, then either set the global feature
   using:
     choco feature enable -name=exitOnRebootDetected
   or pass the option --exit-when-reboot-detected.

Installing the following packages:
fluent-terminal
By installing you accept licenses for the packages.
Progress: Downloading fluent-terminal 0.3.1.0... 100%

fluent-terminal v0.3.1.0 [Approved]
fluent-terminal package files install completed. Performing other installation steps.
The package fluent-terminal wants to run 'chocolateyinstall.ps1'.
Note: If you don't run this script, the installation will fail.
Note: To confirm automatically next time, use '-y' or consider:
choco feature enable -n allowGlobalConfirmation
Do you want to run the script?([Y]es/[N]o/[P]rint): y

Running on: Windows 10 Pro, (Professional), Windows Kernel: 10.0.17763
Extracting C:\ProgramData\chocolatey\lib\fluent-terminal\tools\FluentTerminal.App_0.3.1.0_Test.zip to C:\ProgramData\chocolatey\lib\fluent-terminal\install...
C:\ProgramData\chocolatey\lib\fluent-terminal\install
#< CLIXML
<Objs Version="1.1.0.1" xmlns="http://schemas.microsoft.com/powershell/2004/04"><Obj S="progress" RefId="0"><TN RefId="0"><T>System.Management.Automation.PSCustomObject</T><T>System.Object</T></TN><MS><I64 N="SourceId">1</I64><PR N="Record"><AV>Preparing modules for first use.</AV><AI>0</AI><Nil /><PI>-1</PI><PC>-1</PC><T>Completed</T><SR>-1</SR><SD> </SD></PR></MS></Obj><Obj S="progress" RefId="1"><TNRef RefId="0" /><MS><I64 N="SourceId">1</I64><PR N="Record"><AV>Preparing modules for first use.</AV><AI>0</AI><Nil /><PI>-1</PI><PC>-1</PC><T>Completed</T><SR>-1</SR><SD> </SD></PR></MS></Obj><S S="debug">Host version is 5.1.17763.316, PowerShell Version is '5.1.17763.316' and CLR Version is '4.0.30319.42000'.</S><S S="verbose">Exporting function 'Format-FileSize'.</S><S S="verbose">Exporting function 'Get-ChecksumValid'.</S><S S="verbose">Exporting function 'Get-ChocolateyUnzip'.</S><S S="verbose">Exporting function 'Get-ChocolateyWebFile'.</S><S S="verbose">Exporting function 'Get-EnvironmentVariable'.</S><S S="verbose">Exporting function 'Get-EnvironmentVariableNames'.</S><S S="verbose">Exporting function 'Get-FtpFile'.</S><S S="verbose">Exporting function 'Get-OSArchitectureWidth'.</S><S S="verbose">Exporting function 'Get-PackageParameters'.</S><S S="verbose">Exporting function 'Get-PackageParametersBuiltIn'.</S><S S="verbose">Exporting function 'Get-ToolsLocation'.</S><S S="verbose">Exporting function 'Get-UACEnabled'.</S><S S="verbose">Exporting function 'Get-UninstallRegistryKey'.</S><S S="verbose">Exporting function 'Get-VirusCheckValid'.</S><S S="verbose">Exporting function 'Get-WebFile'.</S><S S="verbose">Exporting function 'Get-WebFileName'.</S><S S="verbose">Exporting function 'Get-WebHeaders'.</S><S S="verbose">Exporting function 'Install-BinFile'.</S><S S="verbose">Exporting function 'Install-ChocolateyDesktopLink'.</S><S S="verbose">Exporting function 'Install-ChocolateyEnvironmentVariable'.</S><S S="verbose">Exporting function 'Install-ChocolateyExplorerMenuItem'.</S><S S="verbose">Exporting function 'Install-ChocolateyFileAssociation'.</S><S S="verbose">Exporting function 'Install-ChocolateyInstallPackage'.</S><S S="verbose">Exporting function 'Install-ChocolateyPackage'.</S><S S="verbose">Exporting function 'Install-ChocolateyPath'.</S><S S="verbose">Exporting function 'Install-ChocolateyPinnedTaskBarItem'.</S><S S="verbose">Exporting function 'Install-ChocolateyPowershellCommand'.</S><S S="verbose">Exporting function 'Install-ChocolateyShortcut'.</S><S S="verbose">Exporting function 'Install-ChocolateyVsixPackage'.</S><S S="verbose">Exporting function 'Install-ChocolateyZipPackage'.</S><S S="verbose">Exporting function 'Install-Vsix'.</S><S S="verbose">Exporting function 'Set-EnvironmentVariable'.</S><S S="verbose">Exporting function 'Set-PowerShellExitCode'.</S><S S="verbose">Exporting function 'Start-ChocolateyProcessAsAdmin'.</S><S S="verbose">Exporting function 'Test-ProcessAdminRights'.</S><S S="verbose">Exporting function 'Uninstall-BinFile'.</S><S S="verbose">Exporting function 'Uninstall-ChocolateyEnvironmentVariable'.</S><S S="verbose">Exporting function 'Uninstall-ChocolateyPackage'.</S><S S="verbose">Exporting function 'Uninstall-ChocolateyZipPackage'.</S><S S="verbose">Exporting function 'Update-SessionEnvironment'.</S><S S="verbose">Exporting function 'Write-ChocolateyFailure'.</S><S S="verbose">Exporting function 'Write-ChocolateySuccess'.</S><S S="verbose">Exporting function 'Write-FileUpdateLog'.</S><S S="verbose">Exporting function 'Write-FunctionCallLogMessage'.</S><S S="verbose">Exporting alias 'Get-ProcessorBits'.</S><S S="verbose">Exporting alias 'Get-OSBitness'.</S><S S="verbose">Exporting alias 'Get-InstallRegistryKey'.</S><S S="verbose">Exporting alias 'Generate-BinFile'.</S><S S="verbose">Exporting alias 'Add-BinFile'.</S><S S="verbose">Exporting alias 'Start-ChocolateyProcess'.</S><S S="verbose">Exporting alias 'Invoke-ChocolateyProcess'.</S><S S="verbose">Exporting alias 'Remove-BinFile'.</S><S S="verbose">Exporting alias 'refreshenv'.</S><S S="verbose">Exporting function 'Format-FileSize'.</S><S S="verbose">Exporting function 'Get-ChecksumValid'.</S><S S="verbose">Exporting function 'Get-ChocolateyUnzip'.</S><S S="verbose">Exporting function 'Get-ChocolateyWebFile'.</S><S S="verbose">Exporting function 'Get-EnvironmentVariable'.</S><S S="verbose">Exporting function 'Get-EnvironmentVariableNames'.</S><S S="verbose">Exporting function 'Get-FtpFile'.</S><S S="verbose">Exporting function 'Get-OSArchitectureWidth'.</S><S S="verbose">Exporting function 'Get-PackageParameters'.</S><S S="verbose">Exporting function 'Get-PackageParametersBuiltIn'.</S><S S="verbose">Exporting function 'Get-ToolsLocation'.</S><S S="verbose">Exporting function 'Get-UACEnabled'.</S><S S="verbose">Exporting function 'Get-UninstallRegistryKey'.</S><S S="verbose">Exporting function 'Get-VirusCheckValid'.</S><S S="verbose">Exporting function 'Get-WebFile'.</S><S S="verbose">Exporting function 'Get-WebFileName'.</S><S S="verbose">Exporting function 'Get-WebHeaders'.</S><S S="verbose">Exporting function 'Install-BinFile'.</S><S S="verbose">Exporting function 'Install-ChocolateyDesktopLink'.</S><S S="verbose">Exporting function 'Install-ChocolateyEnvironmentVariable'.</S><S S="verbose">Exporting function 'Install-ChocolateyExplorerMenuItem'.</S><S S="verbose">Exporting function 'Install-ChocolateyFileAssociation'.</S><S S="verbose">Exporting function 'Install-ChocolateyInstallPackage'.</S><S S="verbose">Exporting function 'Install-ChocolateyPackage'.</S><S S="verbose">Exporting function 'Install-ChocolateyPath'.</S><S S="verbose">Exporting function 'Install-ChocolateyPinnedTaskBarItem'.</S><S S="verbose">Exporting function 'Install-ChocolateyPowershellCommand'.</S><S S="verbose">Exporting function 'Install-ChocolateyShortcut'.</S><S S="verbose">Exporting function 'Install-ChocolateyVsixPackage'.</S><S S="verbose">Exporting function 'Install-ChocolateyZipPackage'.</S><S S="verbose">Exporting function 'Install-Vsix'.</S><S S="verbose">Exporting function 'Set-EnvironmentVariable'.</S><S S="verbose">Exporting function 'Set-PowerShellExitCode'.</S><S S="verbose">Exporting function 'Start-ChocolateyProcessAsAdmin'.</S><S S="verbose">Exporting function 'Test-ProcessAdminRights'.</S><S S="verbose">Exporting function 'Uninstall-BinFile'.</S><S S="verbose">Exporting function 'Uninstall-ChocolateyEnvironmentVariable'.</S><S S="verbose">Exporting function 'Uninstall-ChocolateyPackage'.</S><S S="verbose">Exporting function 'Uninstall-ChocolateyZipPackage'.</S><S S="verbose">Exporting function 'Update-SessionEnvironment'.</S><S S="verbose">Exporting function 'Write-ChocolateyFailure'.</S><S S="verbose">Exporting function 'Write-ChocolateySuccess'.</S><S S="verbose">Exporting function 'Write-FileUpdateLog'.</S><S S="verbose">Exporting function 'Write-FunctionCallLogMessage'.</S><S S="verbose">Exporting alias 'Get-ProcessorBits'.</S><S S="verbose">Exporting alias 'Get-OSBitness'.</S><S S="verbose">Exporting alias 'Get-InstallRegistryKey'.</S><S S="verbose">Exporting alias 'Generate-BinFile'.</S><S S="verbose">Exporting alias 'Add-BinFile'.</S><S S="verbose">Exporting alias 'Start-ChocolateyProcess'.</S><S S="verbose">Exporting alias 'Invoke-ChocolateyProcess'.</S><S S="verbose">Exporting alias 'Remove-BinFile'.</S><S S="verbose">Exporting alias 'refreshenv'.</S><Obj S="information" RefId="2"><TN RefId="1"><T>System.Management.Automation.InformationRecord</T><T>System.Object</T></TN><ToString>Found bundle: C:\ProgramData\chocolatey\lib\fluent-terminal\install\FluentTerminal.App_0.3.1.0_x005F_x86_x005F_x64.appxbundle</ToString><Props><Obj N="MessageData" RefId="3"><TN RefId="2"><T>System.Management.Automation.HostInformationMessage</T><T>System.Object</T></TN><ToString>Found bundle: C:\ProgramData\chocolatey\lib\fluent-terminal\install\FluentTerminal.App_0.3.1.0_x005F_x86_x005F_x64.appxbundle</ToString><Props><S N="Message">Found bundle: C:\ProgramData\chocolatey\lib\fluent-terminal\install\FluentTerminal.App_0.3.1.0_x005F_x86_x005F_x64.appxbundle</S><B N="NoNewLine">false</B><S N="ForegroundColor">DarkYellow</S><S N="BackgroundColor">DarkMagenta</S></Props></Obj><S N="Source">C:\ProgramData\chocolatey\lib\fluent-terminal\install\Install.ps1</S><DT N="TimeGenerated">2019-03-15T10:41:16.8569473-04:00</DT><Obj N="Tags" RefId="4"><TN RefId="3"><T>System.Collections.Generic.List`1[[System.String, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]]</T><T>System.Object</T></TN><LST><S>PSHOST</S></LST></Obj><S N="User">DESKTOP-BK250KQ\harun</S><S N="Computer">DESKTOP-BK250KQ</S><U32 N="ProcessId">11464</U32><U32 N="NativeThreadId">3588</U32><U32 N="ManagedThreadId">13</U32></Props></Obj><Obj S="information" RefId="5"><TNRef RefId="1" /><ToString>Found certificate: C:\ProgramData\chocolatey\lib\fluent-terminal\install\FluentTerminal.App_0.3.1.0_x005F_x86_x005F_x64.cer</ToString><Props><Obj N="MessageData" RefId="6"><TNRef RefId="2" /><ToString>Found certificate: C:\ProgramData\chocolatey\lib\fluent-terminal\install\FluentTerminal.App_0.3.1.0_x005F_x86_x005F_x64.cer</ToString><Props><S N="Message">Found certificate: C:\ProgramData\chocolatey\lib\fluent-terminal\install\FluentTerminal.App_0.3.1.0_x005F_x86_x005F_x64.cer</S><B N="NoNewLine">false</B><S N="ForegroundColor">DarkYellow</S><S N="BackgroundColor">DarkMagenta</S></Props></Obj><S N="Source">C:\ProgramData\chocolatey\lib\fluent-terminal\install\Install.ps1</S><DT N="TimeGenerated">2019-03-15T10:41:17.4194554-04:00</DT><Obj N="Tags" RefId="7"><TNRef RefId="3" /><LST><S>PSHOST</S></LST></Obj><S N="User">DESKTOP-BK250KQ\harun</S><S N="Computer">DESKTOP-BK250KQ</S><U32 N="ProcessId">11464</U32><U32 N="NativeThreadId">3588</U32><U32 N="ManagedThreadId">13</U32></Props></Obj><Obj S="information" RefId="8"><TNRef RefId="1" /><ToString>_x000A_Before installing this app, you need to do the following:</ToString><Props><Obj N="MessageData" RefId="9"><TNRef RefId="2" /><ToString>_x000A_Before installing this app, you need to do the following:</ToString><Props><S N="Message">_x000A_Before installing this app, you need to do the following:</S><B N="NoNewLine">false</B><S N="ForegroundColor">DarkYellow</S><S N="BackgroundColor">DarkMagenta</S></Props></Obj><S N="Source">C:\ProgramData\chocolatey\lib\fluent-terminal\install\Install.ps1</S><DT N="TimeGenerated">2019-03-15T10:41:17.4819763-04:00</DT><Obj N="Tags" RefId="10"><TNRef RefId="3" /><LST><S>PSHOST</S></LST></Obj><S N="User">DESKTOP-BK250KQ\harun</S><S N="Computer">DESKTOP-BK250KQ</S><U32 N="ProcessId">11464</U32><U32 N="NativeThreadId">3588</U32><U32 N="ManagedThreadId">13</U32></Props></Obj><Obj S="information" RefId="11"><TNRef RefId="1" /><ToString>_x0009_- Enable sideloading apps</ToString><Props><Obj N="MessageData" RefId="12"><TNRef RefId="2" /><ToString>_x0009_- Enable sideloading apps</ToString><Props><S N="Message">_x0009_- Enable sideloading apps</S><B N="NoNewLine">false</B><S N="ForegroundColor">DarkYellow</S><S N="BackgroundColor">DarkMagenta</S></Props></Obj><S N="Source">C:\ProgramData\chocolatey\lib\fluent-terminal\install\Install.ps1</S><DT N="TimeGenerated">2019-03-15T10:41:17.4819763-04:00</DT><Obj N="Tags" RefId="13"><TNRef RefId="3" /><LST><S>PSHOST</S></LST></Obj><S N="User">DESKTOP-BK250KQ\harun</S><S N="Computer">DESKTOP-BK250KQ</S><U32 N="ProcessId">11464</U32><U32 N="NativeThreadId">3588</U32><U32 N="ManagedThreadId">13</U32></Props></Obj><Obj S="information" RefId="14"><TNRef RefId="1" /><ToString>_x0009_- Install the signing certificate</ToString><Props><Obj N="MessageData" RefId="15"><TNRef RefId="2" /><ToString>_x0009_- Install the signing certificate</ToString><Props><S N="Message">_x0009_- Install the signing certificate</S><B N="NoNewLine">false</B><S N="ForegroundColor">DarkYellow</S><S N="BackgroundColor">DarkMagenta</S></Props></Obj><S N="Source">C:\ProgramData\chocolatey\lib\fluent-terminal\install\Install.ps1</S><DT N="TimeGenerated">2019-03-15T10:41:17.4819763-04:00</DT><Obj N="Tags" RefId="16"><TNRef RefId="3" /><LST><S>PSHOST</S></LST></Obj><S N="User">DESKTOP-BK250KQ\harun</S><S N="Computer">DESKTOP-BK250KQ</S><U32 N="ProcessId">11464</U32><U32 N="NativeThreadId">3588</U32><U32 N="ManagedThreadId">13</U32></Props></Obj><Obj S="information" RefId="17"><TNRef RefId="1" /><ToString>Enabling sideloading requires user interaction.  Please rerun the script without the -Force parameter.</ToString><Props><Obj N="MessageData" RefId="18"><TNRef RefId="2" /><ToString>Enabling sideloading requires user interaction.  Please rerun the script without the -Force parameter.</ToString><Props><S N="Message">Enabling sideloading requires user interaction.  Please rerun the script without the -Force parameter.</S><B N="NoNewLine">false</B><S N="ForegroundColor">DarkYellow</S><S N="BackgroundColor">DarkMagenta</S></Props></Obj><S N="Source">C:\ProgramData\chocolatey\lib\fluent-terminal\install\Install.ps1</S><DT N="TimeGenerated">2019-03-15T10:41:17.4976036-04:00</DT><Obj N="Tags" RefId="19"><TNRef RefId="3" /><LST><S>PSHOST</S></LST></Obj><S N="User">DESKTOP-BK250KQ\harun</S><S N="Computer">DESKTOP-BK250KQ</S><U32 N="ProcessId">11464</U32><U32 N="NativeThreadId">3588</U32><U32 N="ManagedThreadId">13</U32></Props></Obj></Objs>
0
Only an exit code of non-zero will fail the package by default. Set
 `--failonstderr` if you want error messages to also fail a script. See
 `choco -h` for details.
 The install of fluent-terminal was successful.
  Software installed to 'C:\ProgramData\chocolatey\lib\fluent-terminal\install'

Chocolatey installed 1/1 packages.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
```

It says installation was successful but it is not. Not app called Fluent Terminal is installed.

For manual installation which one should I be using? 

FluentTerminal.App_0.3.1.0_Test.zip or FluentTerminal.App_0.3.1.0_Test_No_Dependencies.zip

Thanks

### PR DESCRIPTION
**Pull request by mikemaccana**
Fixes #230

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ A Terminal Emulator based on UWP and web technologies.
 
 ### Chocolatey package manager installation
 
+- In **Settings**, click **For Developers**, and select **Developer mode**.
```
Comment (skipmeister123) [inline README.md]: Do we need to call out the necessary reboot, or are we content with letting Windows bug them?

Comment (felixse) [reply README.md]: I think Windows will be reboot encouraging enough

**Comment by felixse** [comment]
Thanks 👍

File: README.md (hunk 1 above)
Comment (mikemaccana) [reply README.md]: Weirdly I didn't need to reboot to switch from Sideload to Developer Mode.

**Pull request merged**